lib/HLSL/DxilShaderAccessTracking.cpp

```diff
@@ -75,10 +75,8 @@ RegisterType RegisterTypeFromResourceClass(DXIL::ResourceClass c) {
   case DXIL::ResourceClass::CBuffer: return RegisterType::CBV    ; break;
   case DXIL::ResourceClass::Sampler: return RegisterType::Sampler; break;
   case DXIL::ResourceClass::Invalid: return RegisterType::Invalid; break;
-  default:
-    ThrowIf(true);
-    return RegisterType::Invalid;
   }
+  return RegisterType::Invalid;
```

Comment:
Funfact: MSVC doesn't try to determine if the switch statement contains cases for every possibility, it assumes it doesn't. So if there isn't a return here, it throws an error even though it's technically dead code.